js/reference.js

```diff
@@ -3402,6 +3426,7 @@
                         // 3. invalid self_link (must be not-null and part of a simple key)
                         // 4. invalid aggregate function
```

Comment:
Update this comment with the newly added case when the column is used as part of the iframe input annotation